core/src/main/java/bio/terra/pearl/core/service/datarepo/DataRepoExportService.java

```diff
@@ -3,7 +3,6 @@
 import bio.terra.datarepo.client.ApiException;
 import bio.terra.datarepo.model.JobModel;
 import bio.terra.datarepo.model.JobModel.JobStatusEnum;
-import bio.terra.datarepo.model.TableDataType;
```

Comment:
unrelated cleanup, just noticed it while I was working in this file to reproduce the bug